### FeatureCounts

[Download](http://bioinf.wehi.edu.au/featureCounts/)

[Why featureCounts is better for paired-end](https://www.biostars.org/p/96176/)

```
/home/nazeefa/bin/subread-1.6.2-source/bin/featureCounts -t exon -g gene_id -a /home/nazeefa/dinoflagellates/MMETSP03/Data/counts_matrix/MMETSP0359.nt.fa.transdecoder_dir/longest_orfs.gff3 -o counts.txt /home/nazeefa/dinoflagellates/MMETSP03/Data/assembly/59_bowtie_0/bowtie2_59.bam
```
